homework_12/myproject/homework_model/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.db.models import TextField
+
+# Create your models here.
+
+
+class Person(models.Model):
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
+class Student(Person):
+    db_table = "Student"
```

Comment:
not really a good idea- it's kind of convention of naming tables lowercase and plural (students & teachers)